src/CONST.js

```diff
@@ -790,8 +790,9 @@ const CONST = {
         INVITE: 'invite',
         LEAVE_ROOM: 'leaveRoom',
     },
-    MAX_64BIT_LEFT_HALF: 9223372036,
-    MAX_64BIT_RIGHT_HALF: 854775807,
+    MAX_64BIT_LEFT_PART: 92233,
+    MAX_64BIT_MIDDLE_PART: 7203685,
+    MAX_64BIT_RIGHT_PART: 4775807,
```

Comment:
Maybe a comment would be useful here, as without the context they seem a bit random

Reply:
>without the context they seem a bit random

such a great pun lol